engine/class_modules/monk/sc_monk.cpp

```diff
@@ -468,6 +479,9 @@ void monk_action_t<Base>::consume_resource()
 
     if ( base_t::cost() > 0 )
     {
+      // This triggers prior to cost reduction
+      p()->buff.heart_of_the_jade_serpent->trigger( base_t::cost() );
```

Comment:
This buff is id 456368, and stacks to 2. I believe it's consumed by SotWL and spawns Yu'lon.

Did you mean `buff.heart_of_the_jade_serpent_stack_ww`?